### ISSUE TITLE
[question] How to specify settings for all packages without username/channel?

### ISSUE THREAD

**Issue by Marwan-imverse** [conan-io/conan#14315]
### What is your question?

We have multiple packages developed internally that use each other as requirements, and they also require public packages from conan center.

What I am trying to achieve is, when I build one of our packages in Debug/RelWithDebugInfo, all requirements from our own packages should be in Debug/Debug/RelWithDebugInfo, and all other packages should be in Release. And when I build in Release, everything should be in Release.

I would have added a rule in the profile, but I cannot find a way to force Release for all packages with no user/channel.

Is there a way to do this without complicated CLI gymnastics?

### Have you read the CONTRIBUTING guide?

- [X] I've read the CONTRIBUTING guide

**Comment by memsharded** [comment]
Hi @Marwan-imverse 

Thanks for asking this question.

It is true that there is a gap in the UX to specify packages without user/channel. I am proposing the ``@``-ending syntax to specify this, as we were already doing that in some places in 1.X and 2.0

**Comment by Marwan-imverse** [comment]
Thank you for the answer. In the meantime, isn't there any way to do this while waiting for the upgrade?

**Comment by memsharded** [comment]
I'd try to use the negated approach, define things for packages with user channel with the ``*@*`` pattern, and let the non-prefixed settings be the ones for the packages without user/channel:

```
conan install ... -s build_type=Release -s "*@*:build_type=Debug"
```

**Comment by memsharded** [comment]
https://github.com/conan-io/conan/pull/14338 was merged, it will be in next Conan 2.0.10
